CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # CHANGELOG
 
+## Unreleased
```

Comment:
Note that use use `unclog` to manage unreleased changes.

We should undo the changes to this file. Instead of manually changing `CHANGELOG.md`, we can add an unrelease entry by running something as follows:

> unclog add bug-fixes 1312-fix-gm-stderr --editor vi

[Here](https://github.com/informalsystems/ibc-rs/pull/1329/files#diff-d4737d4f63f51836a82b1bb90846a540d9b467cb25e4790308af887cba6ea7b1) is an example of how the unreleased entry file should look like.